### ISSUE TITLE
File Import Broken

### ISSUE THREAD

**Issue by zedward23** [gaojie058/CollabCoder_system#3]
Txt and Csv files (even the ones in the demo data folder) cannot be loaded into new project pages.

Issue 1: In the function handleConfirm (line 125 in AddNewProjectPage.js), the text variable is not getting populated with the contents of the FileReader(). This means that even before sending the info to the database, the projectObj variable's input_docs value is still empty.

Issue 2: Error Message: uncaughtExceptionError [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client. 
Looks like there are two res.send() calls are happening at the same time (according to stackoverflow) but I can't find where that is. 

In other words, 1. the files are not being properly read/stored and 2. the text of the files are not being sent to the database (MongoDB in my case) either.

My machine is an intel i7 processor on windows 10.

Thank you so much in advance!

**Comment by gaojie058** [comment]
Hi Edward, we have resolved this bug, and please try the latest code. Let us know if the bug remains!
